catalog/ui/search-ui/standard/src/main/webapp/js/module/Tasks.module.js

```diff
@@ -29,7 +29,7 @@ define(['application',
             TaskModule.addInitializer(function() {
 
                 this.subscription = Cometd.Comet.subscribe("/ddf/activities/**", function(resp) {
-                    var task = new Task.Model(resp, {validate: true, parse: true});
+                    var task = new Task.Model(resp.data, {validate: true, parse: true});
```

Comment:
Probably not. I think the logic in the `parse` function could be moved to `initialize`.